UI/LinkText.cpp

```diff
@@ -93,6 +94,51 @@ namespace {
 
         return retval;
     }
+
+    const std::string FOCS_VALUE_TAG_OPEN_PRE("<" + VarText::FOCS_VALUE_TAG);
+    const std::string FOCS_VALUE_TAG_CLOSE("</" + VarText::FOCS_VALUE_TAG + ">");
+    const xpr::sregex FOCS_VALUE_SEARCH = FOCS_VALUE_TAG_OPEN_PRE >> xpr::_s >> (xpr::s1 = REGEX_NON_BRACKET) >> ">" >>
+                                          (xpr::s2 = REGEX_NON_BRACKET) >> FOCS_VALUE_TAG_CLOSE;
+
+    /** Parses VarText::FOCS_VALUE_TAG%s within @p text, replacing value ref name%s with
+     *  the evaluation result of that value ref.  If link label is empty, inserts resolved link argument as label */
+    std::string ValueRefLinkText(const std::string& text, const bool useDescriptionInsteadOfUserString) {
```

Comment:
`CamelCase` for class names but `underscore_separated_lower_case` for variable names